src/draw/DefaultDrawer.js

```diff
@@ -47,10 +45,53 @@ class DefaultDrawer {
      */
     this.resources = resources;
     /**
-     * Margin value between component default position.
+     * Render svg viewbox width.
      * @type {Number}
      */
-    this.margin = 6;
+    this.width = options.width || 1280;
+    /**
+     * Render svg viewbox height.
+     * @type {Number}
+     */
+    this.height = options.height || 1280;
+    /**
+     * Minimum width of a component.
+     * @type {Number}
+     */
+    this.minWidth = options.minWidth || 230;
+    /**
+     * Minimum height of a component.
+     * @type {Number}
+     */
+    this.minHeight = options.minHeight || 50;
+    /**
+     * Padding around components.
+     * @type {Number}
+     */
+    this.padding = options.padding ?? 30;
```

Comment:
Thanks to you, I discovered a new operator !
!! you should use it more (above) !!

Reply:
Agree with Pierre please apply this to all previous properties.

Reply:
Unfortunately, not doable until the doc engine is replaced, since this syntax causes parsing errors